test/parallel/test-fs-long-path.js

```diff
@@ -4,6 +4,11 @@ var fs = require('fs');
 var path = require('path');
 var assert = require('assert');
 
+if (!common.isWindows) {
+  console.log('1..0 # Skipped: this test is Windows-specific.');
+  process.exit();
```

Comment:
@bnoordhuis Ok, done - in 09691de. Please let me know if I should squash the commits now or still wait for more feedback. Thanks.